package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@mtz/mtz-ajax-interceptor",
+  "version": "0.5.0",
+  "description": "A group of elements to support intercepting ajax requests for injecting headers or handling responses globally.",
+  "main": "all-imports.html",
+  "directories": {
+    "example": "demo",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "wct"
```

Comment:
`polymer test` doesn't work for this?  It should be using WCT internally.

Reply:
Nope, it uses the global WCT instance